flooddrake/dynamic_source_term.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import division
+from __future__ import absolute_import
+
+
+def Source(source, t, func):
+    """ Varies the source term in the DG shallow water implementation over time according to a user defined function
+
+        :param source: The source field
+        :type source: :class:`Function'
+
+        :param time: Time
+        :type time: double
+
+        :param func: Time varying function to scale source term with. Input is time, Output scalar.
+        :type func: str
+
+
+    """
+
+    return source.assign(func(t)*source)
```

Comment:
Yes I don't really know why I had that function - I've got rid of it and just inserted that code (dynamic changing of source term) into timestepper. If the checks pass is the new commit ready to merge?